test/test_networks/test_glow.jl

```diff
@@ -306,4 +306,117 @@ dY_ = randn(Float32, size(dY))
 dX_, dθ_, _, _ = G.adjointJacobian(dY_, Y)
 a = dot(dY, dY_)
 b = dot(dX, dX_)+dot(dθ, dθ_)
-@test isapprox(a, b; rtol=1f-3)
\ No newline at end of file
+@test isapprox(a, b; rtol=1f-3)
+
+
+
+
+###########################################Test 3D with split_scales = false #########################
```

Comment:
Is this just copy paste from above? If yes maybe jhust `for N  in [(nx, NY), (nx, NY, nz)]`